akka-actor-typed/src/main/scala/akka/actor/typed/eventstream/EventStream.scala

```diff
@@ -31,6 +31,10 @@ object EventStream {
   /**
    * Subscribe a typed actor to listen for types or subtypes of E
    * by sending this command to the [[akka.actor.typed.ActorSystem.eventStream]].
+   * <p>
+   *   Subscribe several times via send multiple Subscribe(a) are allowed because the
+   * EventStream's watch operation on an Actor is idempotent.
+   * </p>
```

Comment:
The most important part here is that the same actor can create multiple subscriptions to different types of published messages.